view/tabs/memdump.js

```diff
@@ -133,3 +152,69 @@ $("#memdump").on("click", ".dumpline", function() {
         toggleBreakpoint(brkaddr);
     }
 });
+
+
+$("#dumpcontent").on("focusin focusout keyup keydown", ".membytes div[contenteditable]", function(evt) {
+    var $this = $(this);
+    var value = $this.text();
+    switch(evt.type) {
+        case 'keyup':
+        case 'keydown':
+            if(evt.keyCode >= 8 && evt.keyCode <= 9) return; // Backspace, Tab
+            if(evt.keyCode == 13) { // ENTER
+                evt.preventDefault();
+                $this.blur();
+                return;
+            }
+
+            if(!/^[a-fA-F0-9]+$/.test(value)) {
+                // TODO: limit input to valid HEX values
```

Comment:
The emulator can show popup messages with `popout.error()` if you want to inform the user the input is wrong

Reply:
I've added popout.error() and converted invalid values to 0xFF with a warning